Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -69,15 +73,26 @@ def audit
       files = ARGV.resolved_formulae.map(&:path)
     end
 
+    only_cops = ARGV.value("only-cops").to_s.split(",")
```

Comment:
@MikeMcQuaid I updated the code. Simplified cops names. The command syntax now is
`$ brew audit --only-cops=Desc,ComponentsOrder nrpe`
`$ brew audit --except-cops=BottleBlock`
similarly for `brew style` 

I'm thinking unless ilovezfs has another preference we'll keep it this way and later on when most of the audit rules are translated to cops we might think of parsing values from `--only`|`--except`